MapboxNavigation/StyleManager.swift

```diff
@@ -96,6 +96,21 @@ open class StyleManager: NSObject {
         resetTimeOfDayTimer()
     }
     
+    func applyStyle(type styleType: StyleType) {
+        guard currentStyleType != styleType else { return }
+        
+        NSObject.cancelPreviousPerformRequests(withTarget: self, selector: #selector(timeOfDayChanged), object: nil)
+        
+        for style in styles {
+            if style.styleType == styleType {
+                style.apply()
+                currentStyleType = styleType
+                delegate?.styleManager?(self, didApply: style)
+                break
```

Comment:
It's rather unlikely but still possible to split up the styles into something like `[TopBannerDayStyle(), BottomBannerDayStyle()]`, with that in mind, I don't think we should break here.

Reply:
Oh okay, that makes sense. Will do.

Reply:
@frederoni Hmmm... the break can be removed. Since we're only applying a specified `StyleType`, rather than `Style` -- which can be an array, I don't believe it's applicable in this context. Or what do you think?

Reply:
Looks good now 👍